tests/acceptance/testall

```diff
@@ -474,6 +474,9 @@ export CFENGINE_TEST_OVERRIDE_WORKDIR TEMP CFENGINE_TEST_OVERRIDE_EXTENSION_LIBR
 
         TEST_END_TIME=$(unix_seconds)
 
+        # This script does not run under GAINROOT but it needs to write logs
+        $GAINROOT chmod ugo+rwxt "$WORKDIR"
```

Comment:
I thought it wouldn't be necessary, since `WORKDIR` is pre-created. And it's a bit insecure, but probably we don't need to worry about that here.